src/services/API/endpoints.js

```diff
@@ -5,3 +5,5 @@ export const verifyOTPEndpoint = "/verifyotp";
 export const getSessionDataEndpoint = "/getSessionData";
 export const checkBirthdateEndpoint = "/checkBirthdate";
 export const studentSignupEndpoint = "/studentSignup";
+export const createAccessTokenEndpoint =
+  import.meta.env.VITE_APP_PORTAL_BACKEND + "/create-access-token";
```

Comment:
this is an endpoint so only the API endpoint must be mentioned. The base URL can be called from where you making the API call.